### PR TITLE
Vajdalil test environment

### PR DESCRIPTION
**Pull request by vajdalil**

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/KalendaryoApplication.java [hunk 1]
```diff
@@ -4,8 +4,14 @@
...
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootApplication
+@Configuration
+@ActiveProfiles("dev")
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/KalendaryoApplication.java]: `@ActiveProfiles(System.getenv("ACTIVE_PROFILE", "dev"))`

Comment (adamgyulavari) [reply src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/KalendaryoApplication.java]: or use the command line argument version

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/SavingMethodsTestEnvImp.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    @Autowired
+    UserModelRepository userModelRepository;
+
+    @Autowired
+    MergedCalendarRepository mergedCalendarRepository;
+
+    @Override
+    public void saveAuthModel(AuthModel authModel) {
+        if (authModelRepository.findByEmail(authModel.getEmail()) == null) {
```
Comment (adamgyulavari) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/SavingMethodsTestEnvImp.java:30]: @vajdalil awesome! make this whole separation structure one level lower with only one authorize method

**Review by gumimaci88 — DISMISSED** [review]
Let's merge, then perfect it later.

**Review by gumimaci88 — APPROVED** [review]

**Pull request merged**